docs/events/timeline-ui-overview.asciidoc

```diff
@@ -196,8 +196,6 @@ From the *Correlation* tab, you can also do the following:
 [[esql-in-timeline]]
 == Use {esql} to investigate events 
 
-preview::["Do not use {esql} on production environments. This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
-
 NOTE: The {esql} tab is available by default. Since it's in technical preview, you can remove it by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag.
```

Comment:
@michaelolo24 is the feature flag that enables/disables the **ES|QL** Timeline being removed in 8.14? In other words, will the **ES|QL** Timeline tab be available by default when users upgrade to 8.14?

cc: @paulewing

Reply:
Verified with both via Slack that it's ok to remove the note on line 199.

Reply:
Addressed in [69c8aaa](https://github.com/elastic/security-docs/pull/5032/commits/69c8aaa68a48fb97db6e879de1019f47befa8209)